koku/api/tests_query_handler.py

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Test the Report Queries."""
+from api.iam.test.iam_test_case import IamTestCase
+from api.provider.test import create_generic_provider
+from api.report.aws.query_handler import AWSReportQueryHandler
+from api.report.aws.view import AWSInstanceTypeView
+from api.report.test import FakeAWSCostData
+from api.report.test.aws.helpers import AWSReportDataGenerator
+from api.utils import DateHelper
+
+
+class AWSReportQueryTest(IamTestCase):
```

Comment:
Feels like these tests could go here:
https://github.com/project-koku/koku/blob/764cdd03830d487ebb1a34ad69cc0527e6f0e584/koku/api/report/test/aws/tests_queries.py#L42

Reply:
Yes that’s where it previously was located, but Andrew told me we have a 1:1 test-to-python-file organization. these tests are testing the koku/koku/api/query_handler.py file

Reply:
Ok. Then another class name?

Reply:
Ok. Then another class name?

Reply:
Then another class name for the test?

Reply:
Yep will change that

Reply:
renamed to QueryHandlerTest

Reply:
So I could be wrong but per @adberglund  comment https://github.com/project-koku/koku/pull/1552#discussion_r357742705

I think this was suppose to land in koku/koku/api/report/test/aws/tests_queries.py

Right @adberglund ?

Reply:
moved to koku/koku/api/report/test/aws/tests_queries.py as per @chambridge suggestion

Reply:
No you moved this to: koku/api/report/test/tests_queries.py

Reply:
Captured this here (https://github.com/project-koku/koku/pull/1552/files#r368027773) since this is hard to view now the source was removed. and resolving this discussion.

Reply:
oops, it should be moved now. thanks for catching that @chambridge